src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.VisualStudio.Packaging
         new string[] { "SolutionHasProjectCapability:.NET & CPS" }
         )]
 
-    [ProvideMenuResource("Menus.ctmenu", 3)]
+    [ProvideMenuResource("Menus.ctmenu", 4)]
```

Comment:
Why did this number change?

Reply:
It's a version number of the file.

Reply:
Is it tracked externally?

Reply:
Yeah, it's used for caching purposes I believe. I'm not entirely sure how it works TBH.